src/js/buildStandalone.js

```diff
@@ -19,56 +19,56 @@ clientStandaloneRequest.onreadystatechange = function() {
 }
 clientStandaloneRequest.send();
 
+// replace bad characters in user-supplied text
+function safeUser(value) {
+	const regex = /[<>&"'\v]/g;
+	var nv = value;
+	if (nv.match(regex)) {
+		consolePrint(`Unsafe characters found in script will be replaced by !`);
+		nv = nv.replace(regex, '!');
+	}
+	// remove $ for now, they go back later
+	return safeDollar(nv);
+}
+
+// replace $ because it's special in replace
```

Comment:
I don't understand this comment -- open to rework it / remove it.

Maybe it'd be clear enough to name the function more verbosely, like `sanitizeDollarCharacter()`? 
_(And similarly, for `safeUser` above, IMO, though it's a nitpick)_

Reply:
The original comments was
`	// $ has special meaning to JavaScript's String.replace `
so I was sticking somewhere near.

Again, happy to approve a change that makes it easier for the reader.

What are the mechanics to do that?